ramls/scheduled-notice.json

```diff
@@ -17,6 +17,11 @@
       "description": "UUID of related request for request notices",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
     },
+    "userId": {
```

Comment:
Could this be given a name that better reflects the role the user is playing.

When I read the comment, it sounds like this is the recipient user?

Reply:
Yes, makes sense for me. I'm thinking about "recipientId". What do you think, does it reflects the purpose of this field?

Reply:
My personal preference would be `recipientUserId` as that describes both the role and the kind of record we expect the reference to refer to.

Reply:
Agree, changed to "recipientUserId"